x-pack/plugins/ml/public/application/components/annotations/annotation_flyout/index.tsx

```diff
@@ -441,7 +440,22 @@ export const AnnotationFlyout: FC<any> = (props) => {
           </h2>
         </EuiTitle>
       </EuiFlyoutHeader>
-      <AnnotationFlyoutUI {...props} />
+      <AnnotationFlyoutUI {...restProps} />
     </EuiFlyout>
   );
 };
+
+export const AnnotationFlyout: FC<any> = (props) => {
+  return (
+    <MlAnnotationComponent>
+      {(annotationUpdatesService) => {
+        return (
+          <AnnotationFlyoutContainer
+            annotationUpdatesService={annotationUpdatesService}
+            {...props}
+          />
+        );
+      }}
+    </MlAnnotationComponent>
+  );
+};
```

Comment:
I don't think you need this. Simply inject a service instance using react hooks. 
I presume you've looked at `MlTooltipComponent`, but it's a bit different case, as I needed to inject service into class components using render props.

Reply:
Removed the component here https://github.com/elastic/kibana/pull/83216/commits/6eff2c80521ae240e841d21a1bb4581e4ba85786